arches_lingo/src/arches_lingo/components/concept/ConceptLabel/ConceptLabel.vue

```diff
@@ -82,5 +83,11 @@ async function getSectionValue() {
             :resource-instance-id="props.resourceInstanceId"
             :tile-id="props.tileId"
         />
+        <Message
+            v-if="fetchError"
+            severity="error"
+            size="small"
+            >{{ fetchError.message }}
+        </Message>
```

Comment:
I believe this is in the wrong place. it would still show the viewer/editor in addition to the error message. instead it should be:

```
<ProgressSpinner v-if>
<Message v-else-if>
<template v-else>
```

also nit: the message format is a little wonky:

```
        <Message
            v-if="fetchError"
            severity="error"
            size="small"
        >
            {{ fetchError.message }}
        </Message>
```
```